generators/server/index.js

```diff
@@ -1021,6 +1021,16 @@ module.exports = JhipsterServerGenerator.extend({
                     this.copy(SERVER_MAIN_RES_DIR + 'config/cql/changelog/_insert_default_users.cql', SERVER_MAIN_RES_DIR + 'config/cql/changelog/00000000000000_insert_default_users.cql');
                 }
             }
+
+            if (this.applicationType === 'uaa' && !this.fs.exists(SERVER_MAIN_RES_DIR + 'keystore.jks')) {
+                shelljs.mkdir('-p', SERVER_MAIN_RES_DIR);
+                if (shelljs.exec('keytool -genkey -noprompt -keyalg RSA -alias selfsigned -keystore ' + SERVER_MAIN_RES_DIR + 'keystore.jks -storepass password -keypass password -keysize 2048 -dname "CN=Java Hipster, OU=Development, O=' + this.packageName + ', L=, ST=, C="').code !== 0) {
```

Comment:
You could use the yeomans inbuilt spawn command method for this alternatively http://yeoman.io/generator/actions_spawn_command.html